spacepy/empiricals.py

```diff
@@ -120,8 +122,8 @@ def calcLpp(Kpmax, A, B, power=1):
     model_list = ['CA1992', 'M2002', 'RT1970']
 
     if model == 'CA1992':
-        if LT!='all':
-            print('No LT dependence currently supported for this model')
+        if LT != 'all':
+            warnings.warn(RuntimeWarning('No LT dependence currently supported for CA1992 model'))
```

Comment:
```suggestion
            warnings.warn('No LT dependence currently supported for CA1992 model', RuntimeWarning)
```

This is more typical usage

Reply:
Ok, I will change that. I have never done them that way, I have always matched how errors are called e.g. `raise TypeError('windowmean error: winsize/overlap must be timedeltas')`

spacepy doesn't have a lot of warnings but the ones it does match your style.